pkgs/development/python-modules/mat2/paths.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/libmat2/bubblewrap.py b/libmat2/bubblewrap.py
+index 970d5dd..6444e32 100644
+--- a/libmat2/bubblewrap.py
++++ b/libmat2/bubblewrap.py
+@@ -22,11 +22,7 @@ CalledProcessError = subprocess.CalledProcessError
+ 
+ 
+ def _get_bwrap_path() -> str:
+-    which_path = shutil.which('bwrap')
+-    if which_path:
+-        return which_path
+-
+-    raise RuntimeError("Unable to find bwrap")  # pragma: no cover
++    return '@bwrap@'
+ 
+ 
+ def _get_bwrap_args(tempdir: str,
+@@ -37,7 +33,7 @@ def _get_bwrap_args(tempdir: str,
+ 
+     # XXX: use --ro-bind-try once all supported platforms
+     # have a bubblewrap recent enough to support it.
+-    ro_bind_dirs = ['/usr', '/lib', '/lib64', '/bin', '/sbin', '/etc/alternatives', cwd]
++    ro_bind_dirs = ['/nix/store', cwd]
+     for bind_dir in ro_bind_dirs:
+         if os.path.isdir(bind_dir):  # pragma: no cover
+             ro_bind_args.extend(['--ro-bind', bind_dir, bind_dir])
+diff --git a/libmat2/exiftool.py b/libmat2/exiftool.py
+index eb65b2a..6176c15 100644
+--- a/libmat2/exiftool.py
++++ b/libmat2/exiftool.py
+@@ -72,12 +72,4 @@ class ExiftoolParser(abstract.AbstractParser):
+ 
+ @functools.lru_cache()
```

Comment:
This line (`@functools.lru_cache()`) can also be probably removed when the function itself is replaced by `return CONSTANT` stuff. Applies to all these locations which were modified the same way.